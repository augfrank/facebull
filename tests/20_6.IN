M0   C0   C1  2512
M1   C0   C2  2378
M2   C0   C3  2572
M3   C0   C5  2845
M4   C0   C6  2362
M5   C0   C8  2593
M6   C0   C9  2172
M7   C0   C10  2041
M8   C0   C12  2207
M9   C0   C14  2582
M10   C0   C15  2940
M11   C0   C17  2517
M12   C0   C18  2036
M13   C0   C19  2612
M14   C1   C2  2614
M15   C1   C4  2139
M16   C1   C9  2977
M17   C1   C12  2397
M18   C1   C13  2532
M19   C1   C14  2730
M20   C1   C15  2510
M21   C1   C16  2727
M22   C1   C19  2613
M23   C2   C0  2688
M24   C2   C1  2919
M25   C2   C3  2491
M26   C2   C4  2952
M27   C2   C5  2968
M28   C2   C6  2792
M29   C2   C8  2624
M30   C2   C9  2520
M31   C2   C10  2010
M32   C2   C11  2685
M33   C2   C12  2441
M34   C2   C13  2333
M35   C2   C14  2164
M36   C2   C15  2717
M37   C2   C16  2202
M38   C2   C18  2486
M39   C2   C19  2912
M40   C3   C0  2896
M41   C3   C4  2903
M42   C3   C5  2138
M43   C3   C6  2871
M44   C3   C8  2325
M45   C3   C9  2663
M46   C3   C11  2812
M47   C3   C12  2086
M48   C3   C14  2238
M49   C3   C15  2415
M50   C3   C16  2548
M51   C3   C17  2694
M52   C3   C18  2242
M53   C3   C19  2462
M54   C4   C1  2308
M55   C4   C2  2541
M56   C4   C3  2331
M57   C4   C5  2757
M58   C4   C6  2564
M59   C4   C7  2308
M60   C4   C8  2714
M61   C4   C9  2380
M62   C4   C10  2604
M63   C4   C11  2857
M64   C4   C13  2949
M65   C4   C14  2128
M66   C4   C15  2967
M67   C4   C17  2037
M68   C4   C18  2570
M69   C4   C19  2262
M70   C5   C0  2887
M71   C5   C1  2914
M72   C5   C2  2401
M73   C5   C3  2677
M74   C5   C7  2249
M75   C5   C8  2472
M76   C5   C10  2500
M77   C5   C11  2720
M78   C5   C12  2270
M79   C5   C14  2123
M80   C5   C15  2916
M81   C5   C18  2929
M82   C6   C1  2710
M83   C6   C2  2179
M84   C6   C4  2769
M85   C6   C5  2176
M86   C6   C7  2220
M87   C6   C8  2276
M88   C6   C10  2746
M89   C6   C11  2250
M90   C6   C12  2380
M91   C6   C15  2025
M92   C6   C16  2984
M93   C6   C18  2488
M94   C7   C2  2577
M95   C7   C4  2717
M96   C7   C8  2200
M97   C7   C9  2692
M98   C7   C11  2767
M99   C7   C14  2847
M100   C7   C15  2988
M101   C7   C16  2177
M102   C7   C17  2427
M103   C7   C18  2937
M104   C7   C19  2414
M105   C8   C0  2050
M106   C8   C1  2754
M107   C8   C2  2865
M108   C8   C3  2113
M109   C8   C5  2917
M110   C8   C6  2841
M111   C8   C7  2222
M112   C8   C12  2513
M113   C8   C14  2157
M114   C8   C15  2708
M115   C8   C18  2675
M116   C8   C19  2504
M117   C9   C0  2979
M118   C9   C1  2433
M119   C9   C2  2987
M120   C9   C4  2290
M121   C9   C5  2718
M122   C9   C6  2773
M123   C9   C7  2024
M124   C9   C8  2719
M125   C9   C12  2091
M126   C9   C13  2783
M127   C9   C14  2386
M128   C9   C15  2696
M129   C9   C16  2293
M130   C9   C19  2447
M131   C10   C0  2933
M132   C10   C1  2379
M133   C10   C4  2862
M134   C10   C5  2000
M135   C10   C11  2482
M136   C10   C12  2379
M137   C10   C15  2027
M138   C10   C18  2497
M139   C10   C19  2993
M140   C11   C0  2522
M141   C11   C3  2428
M142   C11   C4  2944
M143   C11   C6  2628
M144   C11   C9  2487
M145   C11   C10  2820
M146   C11   C12  2161
M147   C11   C13  2299
M148   C11   C14  2677
M149   C11   C15  2450
M150   C11   C17  2943
M151   C11   C19  2920
M152   C12   C0  2757
M153   C12   C1  2682
M154   C12   C2  2501
M155   C12   C3  2221
M156   C12   C4  2002
M157   C12   C5  2015
M158   C12   C6  2407
M159   C12   C10  2406
M160   C12   C11  2198
M161   C12   C15  2783
M162   C12   C17  2772
M163   C12   C18  2310
M164   C12   C19  2256
M165   C13   C1  2262
M166   C13   C3  2365
M167   C13   C4  2934
M168   C13   C5  2529
M169   C13   C6  2060
M170   C13   C7  2674
M171   C13   C8  2747
M172   C13   C9  2657
M173   C13   C10  2612
M174   C13   C12  2872
M175   C13   C14  2882
M176   C13   C15  2753
M177   C13   C16  2823
M178   C13   C18  2691
M179   C14   C0  2798
M180   C14   C1  2707
M181   C14   C2  2534
M182   C14   C3  2992
M183   C14   C4  2957
M184   C14   C5  2741
M185   C14   C7  2234
M186   C14   C8  2249
M187   C14   C9  2416
M188   C14   C10  2191
M189   C14   C11  2148
M190   C14   C12  2328
M191   C14   C13  2591
M192   C14   C16  2942
M193   C14   C18  2044
M194   C14   C19  2504
M195   C15   C0  2030
M196   C15   C1  2198
M197   C15   C2  2114
M198   C15   C3  2425
M199   C15   C5  2659
M200   C15   C6  2413
M201   C15   C8  2440
M202   C15   C9  2947
M203   C15   C10  2485
M204   C15   C12  2183
M205   C15   C13  2577
M206   C15   C14  2032
M207   C15   C16  2465
M208   C15   C17  2780
M209   C16   C0  2665
M210   C16   C2  2735
M211   C16   C4  2963
M212   C16   C5  2559
M213   C16   C6  2422
M214   C16   C7  2562
M215   C16   C8  2651
M216   C16   C10  2064
M217   C16   C12  2785
M218   C16   C17  2480
M219   C17   C0  2349
M220   C17   C1  2582
M221   C17   C2  2429
M222   C17   C3  2796
M223   C17   C4  2398
M224   C17   C5  2368
M225   C17   C6  2663
M226   C17   C7  2424
M227   C17   C10  2192
M228   C17   C11  2945
M229   C17   C12  2722
M230   C17   C13  2337
M231   C17   C18  2600
M232   C17   C19  2860
M233   C18   C0  2302
M234   C18   C1  2113
M235   C18   C3  2584
M236   C18   C6  2317
M237   C18   C8  2108
M238   C18   C9  2749
M239   C18   C11  2424
M240   C18   C12  2495
M241   C18   C13  2321
M242   C18   C14  2439
M243   C18   C15  2238
M244   C18   C16  2748
M245   C18   C17  2194
M246   C18   C19  2369
M247   C19   C0  2664
M248   C19   C1  2228
M249   C19   C2  2124
M250   C19   C3  2822
M251   C19   C4  2692
M252   C19   C6  2963
M253   C19   C7  2334
M254   C19   C8  2848
M255   C19   C11  2189
M256   C19   C12  2390
M257   C19   C14  2234
M258   C19   C16  2607
M259   C19   C17  2180
