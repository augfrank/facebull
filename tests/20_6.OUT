42609
6 15 37 47 67 79 91 96 108 123 134 147 164 169 193 197 216 219 234 255
